pkg/apis/operators/v1alpha1/servicebinding_types.go

```diff
@@ -112,7 +113,8 @@ type ServiceBinding struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   ServiceBindingSpec   `json:"spec,omitempty"`
+	// +required
+	Spec   ServiceBindingSpec   `json:"spec"`
```

Comment:
is this change really necessary in this PR?

Reply:
Yes, I tried creating SBR without spec and I was successful in doing so. Making services required will only be applicable if the spec is present in SBR if spec itself is not present then Kubernetes will not object to it.

Reply:
Do you feel like I should address it another PR?

Reply:
Either that or change the scope of the PR updating its description. If you decide to keep this change, we should then add additional acceptance tests.

Reply:
I'd say move this to another PR.

Reply:
Okay, I'll create another PR for this.

Reply:
Raised another PR for this: https://github.com/redhat-developer/service-binding-operator/pull/712 PTAL